src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -510,57 +509,64 @@ public function handleError(int $type, string $message, string $file, int $line)
     /**
      * Handles an exception by logging then forwarding it to another handler.
      *
-     * @param \Exception|\Throwable $exception An exception to handle
-     * @param array                 $error     An array as returned by error_get_last()
+     * @param array $error An array as returned by error_get_last()
      *
      * @internal
      */
-    public function handleException($exception, array $error = null)
+    public function handleException(\Throwable $exception, array $error = null)
     {
         if (null === $error) {
             self::$exitCode = 255;
         }
-        if (!$exception instanceof \Exception) {
-            $exception = new FatalThrowableError($exception);
-        }
-        $type = $exception instanceof FatalErrorException ? $exception->getSeverity() : E_ERROR;
+
+        $type = $exception instanceof FatalErrorException ? $exception->getSeverity() : ThrowableUtils::getSeverity($exception);
```

Comment:
same: you could handle `FatalErrorException ` inside `ThrowableUtils::getSeverity`